libr/lang/p/cpipe.c

```diff
@@ -84,3 +99,10 @@ static RLangPlugin r_lang_plugin_cpipe = {
 	.fini = NULL,
 	.run_file = (void*)lang_cpipe_file,
 };
+#else
+#ifdef _MSC_VER
+#pragma message("Warning: cpipe RLangPlugin is not implemented on this platform")
```

Comment:
what about implementing support for windows instead of exposing that this is not supported? the only reason for that is imho the system call to gcc. which may be replaced with the msvc compiler. the rest of the code must be generic and shuold work fine

Reply:
This was never working, as you can see from https://github.com/radareorg/radare2/pull/17198/files#diff-4ff9c2d98fc735f1bc2745b65c380cf4L48 . Honestly, working with windows compiler is a real pain and you also have to support PKG_CONFIG_PATH and LD_LIBRARY_PATH . IMHO this PR is a big *fix*, it doesn't intend to be a new feature, as it would be adding support for windows.

Reply:
I think it's also out of the scope of this PR.